core_lib/data_layers/data/handler/sql_alchemy_data_handler_registry.py

```diff
@@ -54,7 +54,7 @@ def _on_db_session_exit(self, db_session: SqlAlchemyDataHandler):
         else:
             db_session.close()
 
-    def _create_engine(self, config):
+    def _create_engine(self, config) -> create_engine:
         engine = create_engine(build_url(**config.url),
                                pool_recycle=config.session.pool_recycle,
                                echo=config.log_queries)

```

Comment:
sorry my previus comment was not clear 
- there is no need for the `engine` variable, just return from the funciton it self 
- the function return type `engine` 

```
def _create_engine(self, config) -> engine:
    return create_engine(build_url(**config.url),
                                pool_recycle=config.session.pool_recycle,
                                echo=config.log_queries)
```

Reply:
done